.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
I'd probably add the node_modules here too, prevents them from ending up in your PR.